### PR TITLE
Updated monitoring installation

### PR DESCRIPTION
**Pull request by robshelly**
Changes monitoring installation options to the following:

* Monitoring infrastructure - Prometheus, Alertmanager,
Kube-state-metrics
* Monitoring - Service monitors, Prometheus Rules
* Grafana dashboards - Enmasse metrics dashboards

Improved and documented three methods for monitoring installation:

* Bundle
* Ansible
* Manual

### CODE REVIEW

File: documentation/modules/proc-deploy-monitoring-infrastructure.adoc [hunk 1]
```diff
@@ -69,9 +69,11 @@ sed -i s/alertmanager.yaml:*/$(<CONFIG_FILE> | base64 -w0)/g install/alertmanage
...
 ----
 
-. Install using the `monitoring-infrastructure` bundle:
+. Deploy Prometheus, Alertmanager and Kube-state-metrics:
 +
 [options="nowrap",subs="attributes"]
 ----
-{cmdcli} apply -f install/bundles/monitoring
+{cmdcli} apply -f install/components/alertmanager
+{cmdcli} apply -f install/components/promethues
```
Comment (lulf) [inline documentation/modules/proc-deploy-monitoring-infrastructure.adoc]: Typo

File: documentation/modules/proc-deploy-monitoring-infrastructure.adoc [hunk 2]
```diff
@@ -69,9 +69,12 @@ sed -i s/alertmanager.yaml:*/$(<CONFIG_FILE> | base64 -w0)/g install/alertmanage
 {cmdcli} project _{ProductNamespace}_
 ----
 
-. Install using the `monitoring-infrastructure` bundle:
+. Deploy Prometheus, Alertmanager, Grafana operator and Kube-state-metrics:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-monitoring-infrastructure.adoc]:
```suggestion
. Deploy Prometheus, Alertmanager, the Grafana operator, and Kube-state-metrics:
```

File: documentation/modules/proc-deploy-monitoring-using-ansible.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// assembly-monitoring.adoc
+
+[id='deploy-monitoring-ansible-{context}']
+= Deploying monitoring using Ansible
+
+Monitoring can also be deployed during the {Productname}
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-monitoring-using-ansible.adoc]:
```suggestion
Monitoring can also be deployed during the {ProductName} 
```

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 4]
```diff
@@ -30,9 +30,10 @@ The following Ansible configuration settings are supported:
 |register_api_server|Specifies whether to register the API server with {KubePlatform} master|False |no
 |secure_api_server|Specifies whether to enable mutual TLS for the API server|False |no
 |install_example_plans|Specifies whether to install example plans and infrastructure configurations|True|no
-|monitoring_infrastructure|Specifies whether to install full monitoring infrastructure|Not applicable|no
-|service_monitors|Specifies whether to install the service monitors for monitoring {ProductName}|Not applicable|no
-|alerting|Specifies whether to install Prometheus rules that configure alerting for {ProductName}|Not applicable|no
+|monitoring_infrastructure|Specifies whether to install monitoring infrastructure|Not applicable|no
+|monitoring|Specifies whether to install the service monitors and Prometheus rules for monitoring {ProductName}|Not applicable|no
+|grafana_dashboards|Specifies whether to install the Grafana dashboards to graph {ProductName} metrics (Requires link:https://github.com/integr8ly/grafana-operator[Grafana Operator^])|Not applicable|no
+|grafana_operator|Specifies whether to install the Grafana Operator to deploy Grafana|Not applicable|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]: "Grafana operator" ?

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 5]
```diff
@@ -30,9 +30,10 @@ The following Ansible configuration settings are supported:
 |register_api_server|Specifies whether to register the API server with {KubePlatform} master|False |no
 |secure_api_server|Specifies whether to enable mutual TLS for the API server|False |no
 |install_example_plans|Specifies whether to install example plans and infrastructure configurations|True|no
-|monitoring_infrastructure|Specifies whether to install full monitoring infrastructure|Not applicable|no
-|service_monitors|Specifies whether to install the service monitors for monitoring {ProductName}|Not applicable|no
-|alerting|Specifies whether to install Prometheus rules that configure alerting for {ProductName}|Not applicable|no
+|monitoring_infrastructure|Specifies whether to install monitoring infrastructure|Not applicable|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]:
```suggestion
|monitoring_infrastructure|Specifies whether to install the monitoring infrastructure|Not applicable|no
```

**Pull request merged**